docs/content/docs/tree-shaking.tsx

```diff
@@ -0,0 +1,509 @@
+import { ImageZoom } from 'fumadocs-ui/components/image-zoom';
+
+export default function TreeShaking() {
+    return (
+        <figure>
+            <ImageZoom>
+                <svg
```

Comment:
Does it not work to just use the `src` prop of `ImageZoom` and pass it the path to the SVG? My understanding is that `ImageZoom` forwards `src` to `next/image`, which I think is what you want for optimization/performance.

Reply:
It does work but unfortunately that means the SVG no longer have access to the CSS applied to the page. Therefore it makes it impossible to style dynamically for both light and dark modes.